### PR TITLE
[Ide] Added a11y to ProjectFileEntry's "..." button

### PR DESCRIPTION
**Pull request by jstedfast**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/754804

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Components/ProjectFileEntry.cs [hunk 1]
```diff
@@ -43,7 +44,10 @@ public ProjectFileEntry ()
 			entry = new Entry ();
 			PackStart (entry, true, true, 0);
 			
-			button = new Button () { Label = "..." };
+			button = new Button { Label = "..." };
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Components/ProjectFileEntry.cs]: Should this be an ellipsis character?

Comment (jstedfast) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Components/ProjectFileEntry.cs]: Possibly. I figured the safest bet was to change as little as possible.

Comment (Therzok) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Components/ProjectFileEntry.cs]: cc @vancura

Comment (vancura) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Components/ProjectFileEntry.cs]: Yes please. `\u2026` (more at https://gist.github.com/vancura/27fad9793840ceabae14#-ellipsis)

**Comment by jstedfast** [comment]
Thanks Marius :)

**Pull request merged**